src/wallet/rpcwallet.cpp

```diff
@@ -4761,7 +4761,6 @@ extern UniValue importwallet(const JSONRPCRequest& request);
 extern UniValue importprunedfunds(const JSONRPCRequest& request);
 extern UniValue removeprunedfunds(const JSONRPCRequest& request);
 extern UniValue importmulti(const JSONRPCRequest& request);
```

Comment:
nit: Could remove the redundant `extern` as well, since you touch this block of code anyway.